Source/WebCore/dom/NodeList.idl

```diff
@@ -29,5 +29,6 @@
 ] interface NodeList {
     getter Node? item(unsigned long index);
     readonly attribute unsigned long length;
+    readonly attribute boolean isStaticNode;
```

Comment:
This is observable, and we aren't required to expose every method to JS userland code.